server/service.go

```diff
@@ -56,10 +57,17 @@ type BoostService struct {
 	serverTimeouts       HTTPServerTimeouts
 
 	httpClient http.Client
+
+	// Used to stop registerValidatorAtInterval
+	done chan bool
```

Comment:
Another way to implement shutdowns will be to have a context with cancel, and storing the cancel function.

Reply:
I agree, that's what they've been made after all.
Fixed in a2bb054510f72b4545958aa409ad48ffe7700382